guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc

```diff
@@ -13,7 +13,7 @@ For more information, see xref:Creating_a_Custom_File_Type_Repository_{context}[
 For more information, see {AdministeringDocURL}sect-Administering-Installing_the_Katello_Root_CA_Certificate[Installing the Katello Root CA Certificate] in the _Administering {ProjectName}_ guide.
 . An Organization Debug Certificate.
 ifndef::satellite[]
-For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[].
+For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificatee_{context}[].
```

Comment:
We have an extra`e` & the `{context}` depends on the context of the link, whereas the context of the anchor defines the anchor. > We cannot "mix" `{AnyDocURL}` with `{context}`.

Reply:
Ahhhhh
I get it

Reply:
Thank you. I'm tired and was hoping for a quick fix.